pyaedt/rmxprt.py

```diff
@@ -325,3 +329,69 @@ def create_setup(self, name="MySetupAuto", setup_type=None, **kwargs):
         setup.auto_update = True
         setup.update()
         return setup
+
+    @pyaedt_function_handler()
+    def export_configuration(self, output_file):
+        """Export Rmxprt project to config file.
+
+        Parameters
+        ----------
+        output_file : str
+            Full path to json file to be created.
+
+        Returns
+        -------
+        str
+           Full path to json file created.
+        """
+
+        def jsonalize(dict_in, dict_out):
+            dict_out[dict_in.node] = {}
+            for k, v in dict_in.props.items():
+                if not k.endswith("/Choices"):
+                    dict_out[dict_in.node][k] = v
+            for _, c in dict_in.children.items():
+                jsonalize(c, dict_out)
+
+        new_dict = {}
+        jsonalize(self.general.properties, new_dict)
+        jsonalize(self.stator.properties, new_dict)
+        jsonalize(self.rotor.properties, new_dict)
+        jsonalize(self.circuit.properties, new_dict)
+        jsonalize(self.shaft.properties, new_dict)
+        from pyaedt.generic.general_methods import write_configuration_file
+
+        write_configuration_file(new_dict, output_file)
+        return output_file
+
+    @pyaedt_function_handler()
+    def import_configuration(self, input_file):
+        """Parse a json file and assign all the properties to the Rmxprt design.
+
+        Parameters
+        ----------
+        input_file : str
+            Full path to json file to be used.
+
+        Returns
+        -------
+        bool
+            ``True`` when successful, ``False`` when failed.
+        """
+
+        from pyaedt.generic.general_methods import read_configuration_file
+
+        new_dict = read_configuration_file(input_file)
+        for k, v in new_dict.items():
+            dictin = self.stator
```

Comment:
For readability, I would put the line `dictin = self.stator` in an `else:` at the end of the if... else chain.

Reply:
I didn't do because the list of options for stator is the biggest :-D